internal/app/machined/pkg/controllers/network/route_spec.go

```diff
@@ -119,6 +120,17 @@ func (ctrl *RouteSpecController) Run(ctx context.Context, r controller.Runtime,
 	}
 }
 
+// netipPrefixBitsCorrected returns the number of bits in the prefix, corrected for zero value to have bits of 0.
+//
+// Go stdlib returns -1 for zero value, which is not what we want.
+func netipPrefixBitsCorrected(p netip.Prefix) int {
```

Comment:
can we fix our code to be same as expected spec, or am I missing something here?

Reply:
we use it to interface with Linux kernel, and there it expects DstLength to be 0 if the DstAddress is nil.

zero `netip.Prefix` gives us exactly `nil` address, but length (bits in the prefix) became wrong now, as Linux accepts `uint8`, and -1 becomes 255.